datasets/piqa/README.md

```diff
@@ -20,9 +20,58 @@ task_ids:
 - multiple-choice-qa
 paperswithcode_id: piqa
 pretty_name: 'Physical Interaction: Question Answering'
+train-eval-index:
+- config: plain_text
+  task: text-generation
+  task_id: generative_question_answering
```

Comment:
Is this a valid task for autotrain ? I can't find it in https://github.com/huggingface/autonlp-backend/tree/main/zeus/zeus/eval_cards

Reply:
this is the source I'm using, but you are right, it's not supported  yet: https://github.com/huggingface/autonlp-backend/blob/959ea2b53c8b9d75efb1a6bd603d8e3173121985/database/database/enums.py#L102